client/src/app/videos/+video-watch/video-watch.component.ts

```diff
@@ -140,6 +141,11 @@ export class VideoWatchComponent implements OnInit, OnDestroy {
       }, undefined, this.i18n('Subscribe to the account'))
     ]
     if (this.isUserLoggedIn()) this.hotkeysService.add(this.hotkeys)
+
+    if (this.user.pipPlayer) {
```

Comment:
Should be placed in the `if (this.isUserLoggedIn())` block, because you might be unlogged (and thus `this.user` is undefined)

Reply:
at the moment, the popup player is enabled by default. Should I enable it for not logged in users?